Code-Sleep-Python/primitiveBlockChain/darkCoin.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/python3
+# Primitive implementaion of BlockChain Concept
+
+import hashlib as hasher
+import datetime as date
+
+# Block structure
+
+
+class Block():
+    def __init__(self, index, timestamp, data, previous_hash):
+        self.index = index
+        self.timestamp = timestamp
+        self.data = data
+        self.previous_hash = previous_hash
+        self.hash = self.hashBlock()
+
+# Create hash of block
+    def hashBlock(self):
+        sha = hasher.sha256()
```

Comment:
is the original version of blockchain you are implementing?
Can you add an application-specific one here ?

You may get an idea from one of my projects : https://github.com/prateekiiest/Anomaly-Detection-in-BlockChain

Reply:
Few things are missing such as nounce, etc. I tried to make a _chain_ with the block of information. I'll improve it in some time.

Reply:
sure @jadia, this one is promising. Let me know if any updates.